internal/services/springcloud/spring_cloud_gateway_resource.go

```diff
@@ -58,6 +59,13 @@ func resourceSpringCloudGateway() *pluginsdk.Resource {
 				ValidateFunc: validate.SpringCloudServiceID,
 			},
 
+			"addon_json": {
```

Comment:
I think we should be explicit here for the currently supported configuration, not arbitrary JSON. The fact that it is noted to be subject to change in the docs in future is not really any different to any other API parameter. We have known supported values today, and have the ability to update/deprecate etc in future as with any other property or set of properties.

Reply:
Got it, I've removed this `addon_json` field. Please take another look, thanks!